docs/workbenches.md

```diff
@@ -5,61 +5,77 @@ Open Data Hub contains the following workbench images with different variations:
 
-[Installed Packages](https://github.com/opendatahub-io/notebooks/blob/main/jupyter/minimal/ubi9-python-3.9/Pipfile)
+[2023b Packages](https://github.com/opendatahub-io/notebooks/blob/2023b/jupyter/minimal/ubi9-python-3.9/Pipfile) || [2023a Packages](https://github.com/opendatahub-io/notebooks/blob/main/jupyter/minimal/ubi9-python-3.9/Pipfile)
 
 
 ## CUDA
 
-If you are working with compute-intensive data science models that require GPU support, use the Compute Unified Device Architecture (CUDA) notebook image to gain access to the NVIDIA CUDA Toolkit. Using this toolkit, you can optimize your work using GPU-accelerated libraries and optimization tools.
+If you are working with compute-intensive data science models that require GPU support, use the Compute Unified Device Architecture (CUDA) notebook image to gain access to the NVIDIA CUDA Toolkit. You can optimize your work using GPU-accelerated libraries and optimization tools using this toolkit.
+
+[2023b Packages](https://github.com/opendatahub-io/notebooks/blob/2023b/jupyter/minimal/ubi9-python-3.9/Pipfile) || [2023a Packages](https://github.com/opendatahub-io/notebooks/blob/main/jupyter/minimal/ubi9-python-3.9/Pipfile) 
+
+## HabanaAI 
 
-[Installed Packages](https://github.com/opendatahub-io/notebooks/blob/main/jupyter/minimal/ubi9-python-3.9/Pipfile)
+The HabanaAI notebook image optimizes high-performance deep learning (DL) with Habana Gaudi devices. Habana Gaudi devices accelerate DL training workloads and maximize training throughput and efficiency.
+
+[2023b Packages](https://github.com/opendatahub-io/notebooks/blob/main/habana/1.11.0/ubi8-python-3.8/Pipfile) || [2023a Packages](https://github.com/opendatahub-io/notebooks/blob/main/habana/1.10.0/ubi8-python-3.8/Pipfile) 
 
 ## Jupyter Data Science
 
-Use the Standard Data Science notebook image for models that do not require TensorFlow or PyTorch. This image contains commonly used libraries to assist you in developing your machine-learning models. 	
+Use the Standard Data Science notebook image for models that do not require TensorFlow or PyTorch.  
+This image contains commonly used libraries to assist you in developing your machine-learning models. Furthermore, we have integrated several useful libraries and applications. Notably, we've included **Mesa-libgl**, an additional library designed for OpenCV tasks. We've also introduced **Git-lfs**, which provides an efficient solution for handling large files, such as audio samples, videos, datasets, and graphics. The integration of **unixODBC** offers a standardized API for accessing data sources, including SQL Servers and other data sources with ODBC drivers. Lastly, the addition of **Libsndfile** makes it easier to read and write files containing sampled audio data. Additionally, this notebook comes equipped with standard **database clients** for MySQL, PostgreSQL, MSSQL, and MongoDB.
```

Comment:
For all these Jupyter* images - shouldn't we also say that they are based on the JupyterLab IDE? Similarly as we do for the VS Code and R Studio?